shared/base/view.js

```diff
@@ -258,6 +258,15 @@ module.exports = BaseView = Backbone.View.extend({
       };
     }
 
+    // Allow ability to just pass the full "spec" to a lazy loaded view
+    if (this.options.fetch_spec) {
```

Comment:
Yeah, I remember now.  I ran into some trouble with the case and camelcase with rendr-handlebars.  Is there a reason rendr-handlbars is case insensitive?  (Sorry, not directly related to this PR)